user/for-beginners.md

```diff
@@ -1,57 +1,53 @@
 ---
-title: Travis CI for Complete Beginners
+title: Travis CI for Beginners
 layout: en
 permalink: /user/for-beginners/
 ---
 
-If you're not familiar with continuous integration and don't already have a repository that you want to build, this guide will show you what it is all about.
+Welcome to Travis CI! If you are new here or new to Continuous Integration (CI) in general, this doc will provide context and terminology used throughout the platform and documentation. 
 
-You need to sign up for a [GitHub account](https://github.com/) if you do not already have one.
+### What is Continuous Integration (CI)?
```

Comment:
@plaindocs Right... good point. Someone around here should fix that 🙃 
@noloader Sure - makes sense. It does flow better, and it keeps the first sentence from getting too in the weeks of theory vs. implementation...